### PR TITLE
Add dynamic brightness (i.e. dark mode)

### PR DESCRIPTION
**Pull request by ivan-avalos**
Hello! Here's a cool one: dynamic dark/light mode based on system settings! I believe it will only work on iOS for now, not sure if Android has system-wide dynamic dark/light mode as well.

### CODE REVIEW

**Comment by makeworld-the-better-one** [comment]
It'd be nice if there was a toggle setting for this as well.

File: lib/main.dart [hunk 1]
```diff
@@ -34,7 +34,6 @@ void main() async {
     },
     onUpgrade: (db, old, _new) {
       if (old == 1) {
-        db.execute("DROP TABLE hosts");
```
Comment (snoe) [inline lib/main.dart:37]: was this redundant? I was assuming the CREATE would error without the DROP.

**Comment by ivan-avalos** [comment]
Oh, I didn't intend to remove that, it was probably an accident. Let me add it again.

**Comment by snoe** [comment]
Awesome thanks!

**Pull request merged**